contracts/CoveragePool.sol

```diff
@@ -188,6 +188,20 @@ contract CoveragePool is Ownable {
         assetPool.claim(recipient, amountToSeize(portionToSeize));
     }
 
+    /// @notice Grants asset pool shares by minting a given amount of the
+    ///         underwriter tokens for the recipient address. In result, the
+    ///         recipient obtains part of the pool ownership without depositing
+    ///         any collateral tokens.
```

Comment:
Here as well, let's add a note in what circumstances shares are granted.

Reply:
See https://github.com/keep-network/coverage-pools/pull/111/commits/5b3b64029f99da1394f103cc2c7db4f05cdc7966